lib/client.js

```diff
@@ -27,8 +27,11 @@ const { uniqueId, cloneDeep, defaults } = require('lodash');
 const Logger = require('./logger');
 
 const debug = require('debug')('knex:client');
-const debugQuery = require('debug')('knex:query');
+const _debugQuery = require('debug')('knex:query');
 const debugBindings = require('debug')('knex:bindings');
+
+const debugQuery = (sql, txId) => _debugQuery(sql.replace(/%/, '%%'), txId)
```

Comment:
wouldn't that mess up queries that had %% in them already? or that is not a realistic case? maybe we can limit regexp to just single instances of %?

Reply:
I intended this to be `/%/g`. 

Although I cant imagine a situation where we would have multiple consecutive % in an sql query, if we do, I think we would always want them to be reported as many as there are. 

```
> debug('x')('%foo%')
  x NaNoo%

> debug('x')('%%foo%%')
  x %foo% +0ms

> debug('x')('%foo%'.replace(/%/g, '%%'))
  x %foo% +0ms

> debug('x')('%%foo%%'.replace(/%/g, '%%'))
  x %%foo%% +0ms
```

Reply:
fair :)